src/components/felles/chipsknapp.tsx

```diff
@@ -0,0 +1,35 @@
+import { logChips } from '../../utils/logger';
+import { TrashIcon, CheckmarkIcon } from '@navikt/aksel-icons';
+import { Button } from '@navikt/ds-react';
+
+export const LagreChips = (props: { aktiveChips: string[]; lagret: string[] }) => {
+    if (props.lagret !== props.aktiveChips) {
+        return (
+            <Button onClick={() => logChips(props.aktiveChips)} size="small" variant="secondary">
+                Lagre visning
+            </Button>
+        );
+    } else {
+        return (
+            <Button disabled={true} size="small" variant="secondary" icon={<CheckmarkIcon title="a11y-title" />}>
```

Comment:
Dere har sikkert diskutert rundt bruk av "disabled" state, men just-in-case så legger jeg ved denne fra Aksel:

> Disabled
> 
> Disabled state setter hele button-komponenten opacity til 30%. Ved å senke opacity på hele komponenten beholdes det originale uttrykket samtidig som det skaper en deaktivert effekt. Generelt fraråder vi bruk av disabled state fordi det er en dårlig måte å kommunisere med brukerne på. Disabled state bør være siste utvei. [Les mer om hvorfor deaktiverte tilstander er problematiske på Aksel.](https://aksel.nav.no/god-praksis/artikler/deaktiverte-tilstander)

Reply:
Hm, what is solution here @slovrid ?